lib/ecto_factory.ex

```diff
@@ -145,4 +145,9 @@ defmodule EctoFactory do
       {schema} -> {schema, []}
     end
   end
+
+  defp to_keyword_list(map_or_keyword_list) when is_map(map_or_keyword_list) do
+    Enum.map(map_or_keyword_list, fn({key, value}) -> {key, value} end)
+  end
+  defp to_keyword_list(map_or_keyword_list), do: map_or_keyword_list
```

Comment:
We could eliminate this function by using `Enum.into(map_or_keyword_list, [])`

Reply:
👍